src/components/TransferOrderItem.vue

```diff
@@ -31,7 +31,7 @@
       </div>
 
       <div class="qty-progress">
-        <ion-progress-bar :color="getPickedToOrderedFraction(item) > 1 ? 'danger' : 'primary'" :value="getPickedToOrderedFraction(item)" />
+        <ion-progress-bar :color="setProgressBarColor(item)" :value="getPickedToOrderedFraction(item)" />
```

Comment:
We are getting the progressBar color on the basis of some condition, so it can be renamed to something like `getProgessBarColor`

Reply:
Updated the method name to getProgressBarColor.